### PR TITLE
Amazon DynamoDB/v1

### PR DESCRIPTION
**Pull request by jazsmi**
This PR adds documentation for the new Amazon DynamoDB integration.

### CODE REVIEW

File: _database-integrations/dynamodb/dynamodb-v1.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+# -------------------------- #
+#      Replication Info      #
+# -------------------------- #
+
+replication-sections:
+  - title: "Log-based Incremental Replication"
+    anchor: "log-based-incremental-replication"
+    summary: "Details about Log-based Incremental Replication via DynamoDB streams."
+    content: |
+      This integration supports Full Table and Log-based replication. Log-based replication is done through DynamoDB streams, not Amazon Kinesis streams. In order to use streams, they must be enabled on the tables created in DynamoDB. When in your AWS account, go to the `manage streams` section and choose `New Image` or `New and old images`. If one of these aren't selected, the integration will fail during sync. This is handled on a table-by-table basis, so you must enable streams on each.
```
Comment (erinkcochran87) [inline _database-integrations/dynamodb/dynamodb-v1.md]: All of this is for the first paragraph:

> This integration supports Full Table and Log-based replication.

I'd remove this sentence entirely - you're only talking about Log-based here, so no need to mention Full.

---

> Log-based replication is done through DynamoDB streams, not Amazon Kinesis streams.

Try to relate this to the integration itself, and include links to the documentation about the feature:. Ex:

```
Stitch's DynamoDB integration uses [DynamoDB Streams](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html) to perform Log-based Incremental Replication.
```

---

> In order to use streams, they must be enabled on the tables created in DynamoDB.

While true, the phrasing isn't entirely accurate as it relates to the integration. **In order to use Log-based Replication**, streams must be enabled on any table in DynamoDB the user wants to replicate using Log-based Replication. So while streams must be enabled in order to use them, here you're trying to guide the user to understanding that they're enabling them **in this case** so they can use this Replication Method.

So, perhaps: `To use Log-based Incremental Replication, streams must be enabled on every table in DynamoDB you want to replicate using this Replication Method.` This way, you can also get rid of the last sentence in this paragraph (`This is handled on a table-by-table basis, so you must enable streams on each.`) and tighten things up a bit.

---

> When in your AWS account, go to the `manage streams` section and choose `New Image` or `New and old images`. If one of these aren't selected, the integration will fail during sync.

I think this is important and should be called out a bit more - right now it gets lost in the rest of the paragraph. Here's what I'd do:

1. Make this its own paragraph
2. [Link right to the documentation for enabling streams](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html#Streams.Enabling)(meaning you don't have to explain it now)
3. Calling out the options the integration supports. 

So, perhaps:

```
Refer to [Amazon's documentation](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html#Streams.Enabling) for instructions on enabling streams for DynamoDB tables. The **Manage Stream** options must be one of the following, or replication will be unsuccessful:

- `New Image`
- `New and old images`
```

File: _database-integrations/dynamodb/dynamodb-v1.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+ 
+      In AWS, streams are only retained for 24 hours, so when setting up a log-based replication, be sure to set the replication frequency to a time less than 24 hours. If not, you will be at risk of your data aging out.
```
Comment (erinkcochran87) [inline _database-integrations/dynamodb/dynamodb-v1.md]: This is also really important - I'd call more attention to it. Something like:

> **Note**: DynamoDB streams are purged after 24 hours. To ensure you don't lose data, set the integration's Replication Frequency to an interval less than 24 hours (ex: 12 hours).

File: _database-integrations/dynamodb/dynamodb-v1.md [hunk 3]
```diff
@@ -173,10 +173,24 @@ replication-sections:
     anchor: "log-based-incremental-replication"
     summary: "Details about Log-based Incremental Replication via DynamoDB streams."
     content: |
-      This integration supports Full Table and Log-based replication. Log-based replication is done through DynamoDB streams, not Amazon Kinesis streams. In order to use streams, they must be enabled on the tables created in DynamoDB. When in your AWS account, go to the `manage streams` section and choose `New Image` or `New and old images`. If one of these aren't selected, the integration will fail during sync. This is handled on a table-by-table basis, so you must enable streams on each.
+      Stitch's DynamoDB integration uses [DynamoDB Streams](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html) to perform Log-based Incremental Replication. To use Log-based Incremental Replication, streams must be enabled on every table in DynamoDB you want to replicate using this Replication Method.
```
Comment (erinkcochran87) [inline _database-integrations/dynamodb/dynamodb-v1.md]: For external links, please add a `:{target="new"}` attribute at the end of the link. Ex:

```
[Copy that displays on the page](https://www.some-url.com):{target="new"}
```

This opens the link in a new page. It's useful so people don't have to click back a bunch of times when looking through our docs.

File: _database-integrations/dynamodb/dynamodb-v1.md [hunk 4]
```diff
@@ -173,10 +173,24 @@ replication-sections:
...
     content: |
-      This integration supports Full Table and Log-based replication. Log-based replication is done through DynamoDB streams, not Amazon Kinesis streams. In order to use streams, they must be enabled on the tables created in DynamoDB. When in your AWS account, go to the `manage streams` section and choose `New Image` or `New and old images`. If one of these aren't selected, the integration will fail during sync. This is handled on a table-by-table basis, so you must enable streams on each.
+      Stitch's DynamoDB integration uses [DynamoDB Streams](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html) to perform Log-based Incremental Replication. To use Log-based Incremental Replication, streams must be enabled on every table in DynamoDB you want to replicate using this Replication Method. 
+
+      Refer to [Amazon's documentation](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html#Streams.Enabling) for instructions on enabling streams for DynamoDB tables. The **Manage Stream** options must be one of the following, or replication will be unsuccessful:
```
Comment (erinkcochran87) [inline _database-integrations/dynamodb/dynamodb-v1.md]: Same here, re: `:{target="new"}`

File: _database-integrations/dynamodb/dynamodb-v1.md [hunk 5]
```diff
@@ -173,10 +173,24 @@ replication-sections:
...
-      In AWS, streams are only retained for 24 hours, so when setting up a log-based replication, be sure to set the replication frequency to a time less than 24 hours. If not, you will be at risk of your data aging out.
+      Note: DynamoDB streams are purged after 24 hours. To ensure you don't lose data, set the integration's Replication Frequency to an interval less than 24 hours (ex: 12 hours).
+
+  - title: "Full Table Replication"
+    anchor: "full-table-replication"
+    summary: "Details about Full Table Replication Eventually Consistent Reads."
+    content: |
+      To perform Full Table Replications with Stitch's DynamoDB integration, Stitch uses scans to go through tables, rather than queries, so that no user action is required. 
+
+      Stitch's DynamoDB integration will only use eventually consistent reads from your selected DynamoDB tables. What this means for you is that you will not see all of your recent data right away, but it will eventually catch up and return the latest records. For more information on Amazon's DynamoDB Read Consistency, please click [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ReadConsistency.html).
```
Comment (erinkcochran87) [inline _database-integrations/dynamodb/dynamodb-v1.md]: `DynamoDB` > `{{ integration.display_name }}`

File: _database-integrations/dynamodb/dynamodb-v1.md [hunk 6]
```diff
@@ -173,10 +173,24 @@ replication-sections:
...
+        - `New and old images`
  
-      In AWS, streams are only retained for 24 hours, so when setting up a log-based replication, be sure to set the replication frequency to a time less than 24 hours. If not, you will be at risk of your data aging out.
+      Note: DynamoDB streams are purged after 24 hours. To ensure you don't lose data, set the integration's Replication Frequency to an interval less than 24 hours (ex: 12 hours).
+
+  - title: "Full Table Replication"
+    anchor: "full-table-replication"
+    summary: "Details about Full Table Replication Eventually Consistent Reads."
+    content: |
+      To perform Full Table Replications with Stitch's DynamoDB integration, Stitch uses scans to go through tables, rather than queries, so that no user action is required.
```
Comment (erinkcochran87) [inline _database-integrations/dynamodb/dynamodb-v1.md]: `DynamoDB` > `{{ integration.display_name }}`

Comment (erinkcochran87) [inline _database-integrations/dynamodb/dynamodb-v1.md]: But what's a scan? How does it work? Why is it different than a query?

**Comment by jazsmi** [comment]
Hey @cosimon, I finally have some stuff for you to review:

setup steps - [preview](https://deploy-preview-403--stitchdocs.netlify.com/docs/integrations/databases/amazon-dynamodb#setup), [file](https://github.com/stitchdata/docs/pull/403/files#diff-e5291db9dc999162b3131014408e48a6R111)

replication section - [preview](https://deploy-preview-403--stitchdocs.netlify.com/docs/integrations/databases/amazon-dynamodb#replication), [file](https://github.com/stitchdata/docs/pull/403/files#diff-e5291db9dc999162b3131014408e48a6R158)

projection expressions - [preview](https://deploy-preview-403--stitchdocs.netlify.com/docs/integrations/databases/amazon-dynamodb/field-selection-using-projection-expressions), [file](https://github.com/stitchdata/docs/pull/403/files#diff-30f22dc103f8fa9316715d1173fe8781R20)

**Comment by cosimon** [comment]
There are some `{target: "new"}` blocks showing up in the netlify preview.

**Pull request merged**